numba/cuda/tests/cudapy/test_atomics.py

```diff
@@ -876,25 +1118,108 @@ def test_atomic_min_double_shared(self):
 
     def test_atomic_compare_and_swap(self):
         n = 100
-        res = [-99] * (n // 2) + [-1] * (n // 2)
+        fill = -99
+        unfill = -1
+        res = [fill] * (n // 2) + [unfill] * (n // 2)
         random.shuffle(res)
```

Comment:
I think this should be `np.random.shuffle`, as we seed the NumPy RNG in the `setUp` method (this may also be the source of the merge conflict).